Trimble.Modus.Components/Controls/TMInput/MultiLineInput.xaml.cs

```diff
@@ -1,13 +1,18 @@
-namespace Trimble.Modus.Components;
+namespace Trimble.Modus.Components;
+
+using System;
 using Trimble.Modus.Components.Controls.BaseInput;
 
 public partial class MultiLineInput : BaseInput
 {
     #region Bindable Properties
     public static readonly BindableProperty AutoSizeProperty =
-            BindableProperty.Create(nameof(AutoSize), typeof(bool), typeof(MultiLineInput), false, propertyChanged: OnAutoSizePropertyChanged);
+            BindableProperty.Create(nameof(AutoSize), typeof(bool), typeof(MultiLineInput), false, propertyChanged: OnAutoSizePropertyChanged);
+    public new static readonly BindableProperty HeightRequestProperty =
+           BindableProperty.Create(nameof(HeightRequest), typeof(int), typeof(MultiLineInput), propertyChanged: OnIntValueChanged);
```

Comment:
Please change the propertyChanged method name into something meaningful. Perhaps `OnHeightRequestChanged`?

Reply:
Is this done?